plugins/network-elements/juniper-contrail/src/test/java/org/apache/cloudstack/network/contrail/management/MockAccountManager.java

```diff
@@ -24,9 +24,12 @@
 import javax.inject.Inject;
 import javax.naming.ConfigurationException;
 
+import com.cloud.api.auth.SetupUserTwoFactorAuthenticationCmd;
```

Comment:
Yeah, I started the work with that @DaanHoogland but I had to use AccountManager which is in com.cloud and I could not use them from org.apache.cloudstack.api.auth because it is adding circular dependency. 

Any other way/thoughts that I can look at !

Reply:
not now @harikrishna-patnala , as discussed off-line. worrying but not blocking in any way.